src/test/lambda/vue/samInvoke.test.ts

```diff
@@ -90,6 +78,27 @@ describe('Sam Invoke Vue Backend', () => {
         server.panel?.dispose()
     })
 
+    it('can load payload', async function () {
```

Comment:
This tests client + server in unison and should not be considered a unit test anymore. It's closer to a functional test, bordering a full integration test.

There's no easy way to unit test Vue components without pulling in a DOM library since Vue requires the presence of a `window` to function. So we either:
  1. Test components apart of a greater whole (frontend + backend)
  2. Not test components at all, delegating as much logic as possible to the backend
  3. Figure out a way to run testcode within a webview? Seems complicated.

In any case this test is tightly coupled to the frontend's implementation. Success or not is determined largely by inferrence rather than explicit outcomes.